scripts/exrrfs_ioda_bufr.sh

```diff
@@ -7,6 +7,8 @@ cd ${DATA}
 
 # link the prepbufr file
 ${cpreq} ${OBSPATH}/${CDATE}.rap.t${cyc}z.prepbufr.tm00 prepbufr
+${cpreq} ${OBSPATH}/${CDATE}.rap.t${cyc}z.gpsipw.tm00.bufr_d ztdbufr
+${cpreq} ${OBSPATH}/${CDATE}.rap.t${cyc}z.satwnd.tm00.bufr_d satwndbufr
```

Comment:
ztdbufr and satwndbufr files are good to have, but if missing, they should not block the workflow.
So changing `${cpreq}` to 'cp' for line 10 and 11